Apps/AdminWebClient/src/ClientApp/src/views/CovidCard.vue

```diff
@@ -104,7 +104,7 @@ export default class CovidCardView extends Vue {
     }
 
     private get patientName(): string {
-        return `${this.searchResult?.patient?.firstname} ${this.searchResult?.patient?.lastname}`;
+        return `${this.searchResult?.patient?.preferredName.givenName} ${this.searchResult?.patient?.preferredName.surname}`;
```

Comment:
This is AdminWebClient and it calls v2 so I don't think there should be any changes to this code.